### PR TITLE
Create documentation for less

### LINKED ISSUE #100: Documentation for the "less" command

### PR DESCRIPTION
**Pull request by eriwang**
# Advanced HW 9 submission

Closes #100 

Name: Eric Wang
Uniqname: eriwang

### CODE REVIEW

**Comment by mterwill** [comment]
`less +F` is pretty cool, would like to see that included. It's the same idea as `tail -f`.

File: _commands/tools/less.md [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+-------
+
+`less` is used to display contents of a file, as well as allow navigation. Its name is derived from being the _opposite of_ `more`.
+
+~~~ bash
+$ less file.txt
+
+These are the contents of "file.txt".
+
+The output will actually take up the entire screen, and you can navigate using Vim-like commands.
```
Comment (mterwill) [inline _commands/tools/less.md]: Maybe either link to vim or give a brief description of the commands? Maybe briefly dive into `q`, scrolling and `/` for searching?

**Comment by eriwang** [comment]
Thanks for the feedback, Matt! I've also went ahead and added some information on "marking" inside `less`.

**Comment by mterwill** [comment]
Nice, looks great! Thanks, Eric!

**Pull request merged**